DataManagementSystem/Service/StorageElementHandler.py

```diff
@@ -50,6 +51,39 @@
 MAX_STORAGE_SIZE = 0
 USE_TOKENS = False
 
+def getDiskSpace(path, size = 'TB', total = False):
+    """
+      Returns disk usage of the given path.
+      If no size is specified, terabytes will be used by default.
+      If total is set to true, the total disk space will be returned instead.
+    """
+
+    if size.upper() == "KB":
```

Comment:
I would replace this with something like
```python
UNIT_CONVERSION = { "KB" : 1024, "MB" : 1.048576e6, ... }
def getDiskSpace(...):
  size_to_convert = size.upper()
  if size_to_convert not in UNIT_CONVERSION:
    return S_ERROR( "no valid size" )
  convert = UNIT_CONVERSION[size_to_convert]
```

This is shorter, imo easier to understand, debug/maintain and in general better coding style. There's probably even better solutions (I don't like the total flag) but that should be ok.